modules/cli/src/test/java/org/apache/ignite/cli/commands/UrlOptionsNegativeTest.java

```diff
@@ -64,28 +69,45 @@ private void setUp(Class<?> cmdClass) {
         setUp(cmdClass);
         List<String> options = new ArrayList<>();
-        options.add("--" + urlOptionName + "=" + urlOptionValue);
+        options.add(urlOptionName + urlOptionValue);
         options.addAll(additionalOptions);
         exitCode = cmd.execute(options.toArray(new String[0]));
     }
 
     static List<Arguments> cmdClassAndOptionsProvider() {
         return List.of(
-                Arguments.arguments(NodeConfigShowSubCommand.class, "node-url", List.of()),
-                Arguments.arguments(NodeConfigUpdateSubCommand.class, "node-url", List.of("config")),
-                Arguments.arguments(ClusterConfigShowSubCommand.class, "cluster-url", List.of()),
-                Arguments.arguments(ClusterConfigUpdateSubCommand.class, "cluster-url", List.of("config"))
+                Arguments.arguments(NodeConfigShowSubCommand.class, "--node-url=", List.of(), false),
+                Arguments.arguments(NodeConfigUpdateSubCommand.class, "--node-url=", List.of("{key: value}"), false),
+                Arguments.arguments(ClusterConfigShowSubCommand.class, "--cluster-url=", List.of(), false),
+                Arguments.arguments(ClusterConfigUpdateSubCommand.class, "--cluster-url=", List.of("{key: value}"), false),
+                Arguments.arguments(NodeConfigShowReplSubCommand.class, "--node-url=", List.of(), true),
+                Arguments.arguments(NodeConfigUpdateReplSubCommand.class, "--node-url=", List.of("{key: value}"), true),
+                Arguments.arguments(ClusterConfigShowReplSubCommand.class, "--cluster-url=", List.of(), true),
+                Arguments.arguments(ClusterConfigUpdateReplSubCommand.class, "--cluster-url=", List.of("{key: value}"), true),
+                Arguments.arguments(ConnectCommand.class, "", List.of(), true)
+        // TODO https://issues.apache.org/jira/browse/IGNITE-17091
+        //                Arguments.arguments(StatusCommand.class, "--cluster-url=", List.of()),
+        // TODO https://issues.apache.org/jira/browse/IGNITE-17102
+        //                Arguments.arguments(ClusterShowCommand.class, "--cluster-url=", List.of()),
+        // TODO https://issues.apache.org/jira/browse/IGNITE-17092
+        //                Arguments.arguments(TopologyCommand.class, "--cluster-url", List.of()),
+        // TODO https://issues.apache.org/jira/browse/IGNITE-17162
+        //                Arguments.arguments(ClusterCommandSpec.InitClusterCommandSpec.class, "---cluster-url=",
+        //                        List.of("--cluster-name=cluster", "--meta-storage-node=test"))
         );
     }
 
     @ParameterizedTest
     @MethodSource("cmdClassAndOptionsProvider")
     @DisplayName("Should display error when wrong port is given")
-    public void incorrectPortTest(Class<?> cmdClass, String urlOptionName, List<String> additionalOptions) {
+    void incorrectPort(Class<?> cmdClass, String urlOptionName, List<String> additionalOptions, boolean isReplCommand) {
```

Comment:
Just split this test into two repli and non-repl. Please, don't use flags like `isReplCommand`